parley/src/style/mod.rs

```diff
@@ -142,7 +142,7 @@ pub enum StyleProperty<'a, B: Brush> {
     /// Extra spacing between letters.
     LetterSpacing(f32),
     /// Control over where words can wrap.
-    WordBreak(WordBreakStrength),
+    WordBreak(LineBreakWordOption),
```

Comment:
It'd be nice to add our own type for this (called `WordBreak` to match the CSS property) just to avoid exposing `icu` types directly allowing us to update those dependencies without a breaking change.